### ISSUE TITLE
NotHaveSameCount

### ISSUE THREAD

**Issue by jnyrup** [fluentassertions/fluentassertions#597]
There does not seem to be support for the negation of HaveSameCount(IEnumerable otherCollection).

I propose addition of NotHaveSameCount(IEnumerable otherCollection).

If the proposal is welcomed, I'll be happy to submit a PR once 5.0 is merged into master (unless somebody beats me to it).

**Comment by dennisdoomen** [comment]
See #596
